.github/workflows/defer-integration-tests.yml

```diff
@@ -42,3 +42,31 @@ jobs:
           DEFER_WITH_ROUTER_TESTS: true
         run: |
           ./gradlew --no-daemon --console plain -p tests :defer:allTests
+  defer-with-apollo-server-tests:
+    runs-on: ubuntu-latest
+    if: github.repository == 'apollographql/apollo-kotlin'
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 #v4.1.7
+
+      - name: Install and run graph
+        working-directory: tests/defer/apollo-server/
+        run: |
+          npm install --legacy-peer-deps
+          npx patch-package
+          APOLLO_PORT=4000 npm start &
+
+      - name: Setup Java
+        uses: actions/setup-java@99b8673ff64fbf99d8d325f52d9a5bdedb8483e9 #v4.2.1
+        with:
+          distribution: 'temurin'
+          java-version: 17
```

Comment:
Any thoughts about removing this? I'm trying to keep our yaml files clean (see https://github.com/apollographql/apollo-kotlin/pull/6705#discussion_r2375537148)

Reply:
No objection either 👍